game/menu/gamemanager.gd

```diff
@@ -1,7 +1,14 @@
 extends Node
 
-var _games = []
+## type of GAME_DATA_CACHE: Dictionary[String, Dictionary]
+##                                  (file name) (the data)
+const GAME_DATA_CACHE := {}  # NEVER MODIFY THIS, IF YOU DON'T KNOW WHAT YOU ARE DOING!
```

Comment:
Not really sure about all these. `ALL_UPPERCASE` should ideally really be restricted to actual constant and never changing expressions.

Reply:
The linter doesn't like it when it's a const and not upper case. And I agree with that.